### DISCUSSION TITLE
Normal Test Directory set to null

### DISCUSSION THREAD

**Discussion by jonthierry** [open-edge-platform/anomalib#1619] [Q&A]
Howdy, 

I've been using Anomalib for quite a long time now, to the point I start to analyze the little things. How does the model get normal test images if the #normal_test_dir flag is set to null?

**Comment by willyfh** [comment]
Hi @jonthierry , as per my understanding, it will samples the normal images from the training set based on the `test_split_ratio`, as long as we don't set the `test_split_mode` to `none`. You can check the logic here: https://github.com/openvinotoolkit/anomalib/blob/0709caaeeb8df1a6ed69818446568f12b8e5fc87/src/anomalib/data/base/datamodule.py#L129-L137
